vshard/router/init.lua

```diff
@@ -806,7 +849,7 @@ local function router_map_callrw(router, func, args, opts)
         timeout = consts.CALL_TIMEOUT_MIN
     end
     local deadline = fiber_clock() + timeout
-    local err, err_id, res, ok, map
+    local err, err_uuid, res, ok, map
```

Comment:
Please, don't. The term ID is used on purpose, because in the new vshard version we support identification by names, not only by UUIDs.

Reply:
Fixed